src/views/new/Post.vue

```diff
@@ -74,7 +74,7 @@
                   </div>
                   <div class="btn-chevron">
                     <icon
-                      :icon="dropdownOpen ? 'arrow-up' : 'arrow-down'"
+                      :icon="dropdownOpen ? 'arrow-up' : 'arrow-down-editor'"
```

Comment:
Same in [here](https://github.com/NCTU-SYNC/sync-frontend/pull/68/files#diff-50df14f911a18b1b7d7218e618cd18e41efbc4424c1f787fc908e0c3f238bc68R34)